lerobot/configs/default.yaml

```
@@ -10,6 +10,9 @@ hydra:
     name: default
 
 device: cuda  # cpu
+# `use_amp` determines whether to use Automatic Mixed Precision (AMP) for training and evaluation. With AMP,
+# automatic gradient scaling is used.
+use_amp: false
```

Should we set it to true by default?
```suggestion
use_amp: true
```

---

Maybe in a second PR once all our models on the hub are fp16 checkpoints?

---

Yes I agree we should do it next. Good suggestion.